querybook/webapp/components/DataDoc/DataDocDeletePreview.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+
+import { IDataCell } from 'const/datadoc';
+import { ThemedCodeHighlightWithMark } from 'ui/CodeHighlight/ThemedCodeHighlightWithMark';
+
+interface IProps {
+    cell: IDataCell;
+}
+
+export const DataDocDeletePreview: React.FunctionComponent<IProps> = ({
```

Comment:
probably don't need another component for this

Reply:
Responding to this comment and the previous comment, the separate component was added because without it CodeMirror wouldn't show the query contents when the delete confirmation first loaded. The fix we found was to force it to rerender by changing the state variable with useEffect, which isn't possible without a separate component.